inc/osvr/ClientKit/Imaging_decl.h

```diff
@@ -50,23 +49,19 @@ namespace clientkit {
     /// @brief The user-friendly imaging report. Note that passing this around
     /// by value is OK (doesn't copy the image), and the easiest way to ensure
     /// that the image buffer doesn't get freed before you're done using it.
-    struct ImagingReportOpenCV {
+    struct ImagingReport {
         /// @brief The device sensor number this frame came from.
         OSVR_ChannelCount sensor;
 
-        /// @brief An OpenCV Mat header for accessing the image data in the
-        /// buffer. Does not store the data internally (to reduce copies)!
-        cv::Mat frame;
-
```

Comment:
There is no lifetime management here, the `cv::Mat` explicitly specifies that it will not de-allocate the data buffer if it was initialized with a data pointer.